languages/elixir/exercises/concept/charlists/.docs/introduction.md

```diff
@@ -0,0 +1,48 @@
+Charlists are created using single quotes.
+
+```elixir
+'hello'
+```
+
+A charlist is a list of integers where all the integers are valid code points.
```

Comment:
```suggestion
A charlist is a list of integers where all the integers are the unicode decimal value of each character: known as a code point.
```

Reply:
👍, but I avoided the word "decimal" because you can as well write the code points as hexadecimal numbers.